spec/features/transactions_filter_spec.rb

```diff
@@ -254,13 +254,13 @@
 
     context 'when custom' do
       let!(:transaction)  { Timecop.travel(2013,12,12) {
-        create :transaction, bank_account: ba } }
+        create :transaction, bank_account: ba, date: Time.now } }
```

Comment:
@yunixon because you should use in factory: `date: { Time.now }`. Read about lazy attributes in factory girl. https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#lazy-attributes